### PR TITLE
Upgraded Storybook to latest

### PR DESCRIPTION
**Pull request by christian314159**
## Why
- New versions are released, old ones have bugs

## What
- Upgrading versions

### CODE REVIEW

**Review by gyfchong — APPROVED** [review]

File: storybook/main.ts [hunk 1]
```diff
@@ -42,11 +49,12 @@ const config = {
...
         const isHTMLElementProp =
           prop.parent?.fileName.includes("node_modules/@types/react") ?? false
-
         return !isHTMLElementProp
       },
     },
   },
+  docs: {
+    autodocs: true,
+  },
```
Comment (HeartSquared) [inline storybook/main.ts:58]: We don't want this :)

**Review by HeartSquared — CHANGES_REQUESTED** [review]

File: storybook/main.ts [hunk 2]
```diff
@@ -20,16 +20,23 @@ const getStoryPathsFromEnv = (): string[] | false => {
...
 ]
-
 const config = {
   stories: getStoryPathsFromEnv() || defaultStoryPaths,
-  addons: ["@storybook/addon-essentials", "@storybook/addon-a11y"],
-  staticDirs: [{ from: "./assets", to: "/static/media" }],
+  addons: [
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y",
+    "@storybook/addon-mdx-gfm",
```
Comment (HeartSquared) [inline storybook/main.ts:32]: Pretty sure we don't need this either

Comment (christian314159) [reply storybook/main.ts:32]: I assume you specifically mean the `@storybook/addong-mdx-gfm`? I think it was added during the update flow, but I can remove it again (I hope w/o problems ;) )

Comment (HeartSquared) [reply storybook/main.ts:32]: Yeah we don't need this plugin added by the migration :)

File: storybook/main.ts (hunk 1 above)
Comment (HeartSquared) [reply storybook/main.ts:58]: I wonder if we should be explicit and put `autodocs: "tag"` (or whatever it should be) in to stop the auto migrate prompt next time 🤔

Comment (HeartSquared) [reply storybook/main.ts:58]: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#autodocs-changes

>However, as of 7.0, the new default is 'tag'

Comment (christian314159) [reply storybook/main.ts:58]: Ok, I'm then just going to remove the setting, as it will default to `tag`.

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**